### DISCUSSION TITLE
Multi Language

### DISCUSSION THREAD

**Discussion by ghost** [thedevdojo/voyager#5435] [Q&A]
Multilanguage Issues: Unable to get model language.

Dear Team, Please can anyone help with the translation. I've manage to add and test multi lang but for model am facing issues

My Model:-

![Screen Shot 2021-08-25 at 4 38 57 PM](https://user-images.githubusercontent.com/35699210/130791896-e926763d-ade9-4f0c-9221-722f268dddae.png)

My Controller:-

![Screen Shot 2021-08-25 at 4 39 26 PM](https://user-images.githubusercontent.com/35699210/130791958-b2e000da-394c-4d07-94ef-9243b5871f9f.png)

My View:-

![Screen Shot 2021-08-25 at 4 39 55 PM](https://user-images.githubusercontent.com/35699210/130792037-8eb0a2ec-3c4b-4e88-ae13-1552e4496454.png)

**Comment by ghost** [comment]
@MrCrayon Please can you help me

**Reply by ghost** [reply]
Hi @fletch3555 humbly request your help if possible on this issue. May be am doing something wrong please help

**Comment by MrCrayon** [comment]
What's the output of `$item->btn_name` and what are you expecting?

**Reply by ghost** [reply]
The output i was looking for it to get both en & fr language inside my views.

I've got a solution i don't know if this is the correct way of doing it.

Inside my foreach i've added 

@php
    if (Voyager::translatable($item)) {
        $item = $item->translate($item->locale);
    }
@endphp

This worked for me.
